draft-ietf-dnsop-svcb-https.md

```diff
@@ -41,9 +41,9 @@ informative:
 
 This document specifies the "SVCB" and "HTTPS" DNS resource record (RR)
 types to facilitate the lookup of information needed to make connections
-for origin resources, such as for HTTPS URLs.  SVCB records
-allow an origin to be served from multiple network
-locations, each with associated parameters (such as transport protocol
+to network services, such as for HTTPS URLs.  SVCB records
```

Comment:
```suggestion
to network services, such as for HTTPS origins.  SVCB records
```
Not text changed by this PR, but I'd probably talk about the HTTPS origin as one of the services being described, rather than the URL.  In fact, this is explicitly stated further down.